subsys/bluetooth/shell/bt.c

```diff
@@ -193,27 +193,38 @@ static bool data_cb(struct bt_data *data, void *user_data)
 	}
 }
 
-static void print_data_set(uint8_t type, uint8_t set_value_len,
+static void print_data_hex(const uint8_t *data, uint8_t len, bool reverse)
+{
+	shell_fprintf(ctx_shell, SHELL_NORMAL, "0x");
+	for (uint8_t i = 0; i < len; i++) {
+		shell_fprintf(ctx_shell, SHELL_NORMAL, "%02x", data[reverse ? len-1-i : i]);
+	}
+}
+
+static void print_data_set(uint8_t type, uint8_t initial_value_len, uint8_t set_value_len,
 			   const uint8_t *scan_data, uint8_t scan_data_len)
 {
-	uint8_t min_value_len = MIN(set_value_len, scan_data_len);
+	uint8_t idx = 0;
 
-	shell_fprintf(ctx_shell, SHELL_INFO, "%*sType 0x%02x: ",
-		      strlen(scan_response_label), "",
-		      type);
-	for (uint8_t i = 0U; i < (scan_data_len / min_value_len); i++) {
-		if (i > 0L) {
-			shell_fprintf(ctx_shell, SHELL_INFO, ", ");
-		}
+	shell_fprintf(ctx_shell, SHELL_NORMAL, "%*sType 0x%02x: ",
```

Comment:
I like this output to stand out while using the shell manually. I can see this may be painful when parsing shell output programmatically. Can we somehow make a configuration option for this instead?

Reply:
I'll see if I can use the shell color toggle

Reply:
colors can be switched off with `shell colors off` - so reintroduced the info colors